Add new Bigquery parameter (#7821)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/bigquery-configs.md

| `compute_region`        | `<string>`  | no       | ``        | `<COMPUTE_REGION>` |
| `gcs_bucket`            | `<string>`  | no       | ``        | `<GCS_BUCKET>` |
| `packages`              | `<string>`  | no       | ``        | `['numpy<=1.1.1', 'pandas', 'mlflow']` |

> | `enable_change_history` | `<boolean>` | no       | ``        | `True`, `False`   |

Author: Is the default value correct?

- The `gcs_bucket` parameter
  - The `gcs_bucket` parameter specifies the GCS bucket used for storing artifacts for the job.

- The `enable_change_history` parameter

>   - The `enable_change_history` parameter enables [BigQuery's change history feature](https://cloud.google.com/bigquery/docs/change-history) which tracks changes made to a BigQuery table. When enabled, users can audit and debug the behavior of their incremental models.

Reviewer: Suggest using second person perspective
```suggestion
  - The `enable_change_history` parameter enables [BigQuery's change history feature](https://cloud.google.com/bigquery/docs/change-history) which tracks changes made to a BigQuery table. When enabled, you can use the change history to audit and debug the behavior of your incremental models.
```
